GVFS/GVFS.FunctionalTests/Tools/GitHelpers.cs

```diff
@@ -126,7 +145,20 @@ public static void CheckGitCommandAgainstGVFSRepo(string virtualRepoRoot, string
             int resetTimeout = Timeout.Infinite,
             bool skipReleaseLock = false)
         {
-            return RunCommandWithWaitAndStdIn(enlistment, resetTimeout, LockHolderCommandPath, skipReleaseLock ? "--skip-release-lock" : string.Empty, GitHelpers.LockHolderCommandName, "done", out processId);
+            string args = LockHolderCommandPath;
+            if (skipReleaseLock)
+            {
+                args += " --skip-release-lock";
+            }
+
+            return RunCommandWithWaitAndStdIn(
+                enlistment,
+                resetTimeout,
+                "dotnet",

```

Comment:
So we are running the `dotnet` command and passing the lock holder as an arg?  Why can't we just run the lockholder?

Reply:
This is because of the conversation in the other comment, Windows does not have a `dotnet publish` step.

Reply:
after the migration to .netcore, the extension now is `.dll` and became easier to launch it using `dotnet`